### DISCUSSION TITLE
routing.xml and less frequent tags

### DISCUSSION THREAD

**Discussion by tbulligan** [osmandapp/OsmAnd#18478] [Maps Related]
Hi. I'm making progress implementing navigation for light quadricycles (category L6e), but I'm missing one key component: the recognition of the `"microcar = no"` tag. In the Netherlands, that's the tag we use for roads that are marked with the C9 sign explicitly forbidding access to that vehicle category.

Under `way attribute="access">`, these lines successfully exclude roads that are by default forbidden to light quadricycles.

    <select value="-1" t="highway" v="motorway"/>
    <select value="-1" t="highway" v="motorway_link"/>
    <select value="-1" t="highway" v="trunk"/>
    <select value="-1" t="highway" v="trunk_link"/>
    <select value="-1" t="motorroad" v="yes"/>

I hoped that this line below would see the `"microcar = no"` tag but it doesn't.

    <select value="-1" t="microcar" v="no"/>

Does OsmAnd skip some tags when extracting map data from OpenStreetMap?

If so, how can I configure its parser to keep the `microcar` tag?

**Comment by vshcherb** [comment]
All tags for binary maps are located in https://github.com/osmandapp/OsmAnd-resources/blob/master/obf_creation/rendering_types.xml
